sdks/tutorial/petra/foreword.md

```diff
@@ -0,0 +1,29 @@
+# Petra: The Tutorial
+
+Welcome to iCure! If you haven't had the chance to read the [Introduction](../../intro.md) yet, we invite you to do so before continuing to learn more about what iCure is and specifically what iCure MedTech is.
+
+In this tutorial, we will show you how to create a medical application with iCureMedTech for patients. Specifically, we will create a menstrual cycle tracking application. This application will allow patients to add menstrual cycle tracking data, view it, and optionally share it with their doctor.
+
+Given the nature of this tutorial, we will only focus on the features of the application and not on its design. A Github repository is available with all the frontend code as well as the different visuals you will need during this tutorial.
+
+## Callouts
+
+You will encounter callouts throughout this tutorial to draw your attention to different important points:
+
+:::tip
+These callouts are here to help you understand what you are doing and why you are doing it. So, they should be read carefully.
+:::
+
+:::caution
+These callouts are here to warn you of certain important points. So, they should be read carefully.
+:::
+
+:::danger
+These callouts are here to warn you of certain important points. So, they should be read carefully.
```

Comment:
```suggestion
These callouts notify you about potentially dangerous information about the current topic. You need to read them carefully in order to avoid any issue during this tutorial.
```